.github/ISSUE_TEMPLATE/5-flaky-test.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "\U0001F4AB Found a flaky test?"
+about: Please provide information about the failure.
+
+---
+<!--
+Thank you for reporting a possible flaky test in Node.js.
+
+Please fill in as much of the template below as you can.
+
+-->
+* test file - [NAME_OF_TEST_WITH_SUITE (e.g. parallel/test-https)](https://github.com/nodejs/node/blob/master/test/NAME_OF_TEST_WITH_SUITE.js)
+* ci job  - [CI_JOB_DESCRIPTOR (e.g. aix/21613)](https://ci.nodejs.org/job/node-test-CI_JOB_DESCRIPTOR)
+* ci worker - [CI_WORKER_NAME](https://ci.nodejs.org/computer/CI_WORKER_NAME/)
+* output:
+  ```
+  CI console output for the failing test.
+  Or content of JUnit test details page.
+  Note: should be indented with 2 spaces to align with * bullet point.
```

Comment:
I'd remove this. Having to indent output by two spaces can be annoying/tedious. It's OK if it's not indented (or does that cause a big problem--seems mostly aesthetics)?